sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/MigrationGuide.md

```diff
@@ -0,0 +1,111 @@
+# Migrate from Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker to Azure.AI.Language.QuestionAnswering
+
+This guide is intended to assist in the migration to version 1 of the Question Answering client library [`Azure.AI.Language.QuestionAnswering`](https://www.nuget.org/packages/Azure.AI.Language.QuestionAnswering) from version 2 of [`Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker`](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker). It will focus on side-by-side comparisons for similar operations between the two packages.
```

Comment:
I had to read this a couple times because the "version 1" and "version 2" were confusing. Since we're starting over with version 1 but a completely different package name, I think we should leave out the versions.

@annatisch @mshaban-msft @ZulaMostafa what do you think? (Since the .NET and Python migration guides will read about the same, sans code.)

Reply:
Agreed, specially that they are mentioned with details somewhere else in the migration guide.